Alignment/OfflineValidation/bin/PVtrends.cc

```diff
@@ -152,17 +153,17 @@ int trends(int argc, char* argv[]) {
       assert(hRMS  != nullptr);
       
       TString gtitle = alignment.second.get<string>("title");
-      //gMean->SetTitle(gtitle); // for the legend
-      gMean->SetTitle(""); // for the legend
-      gMean->SetMarkerSize(0.6);
+      gMean->SetTitle(gtitle); // for the legend
+      //gMean->SetTitle(""); // for the legend
+      gMean->SetMarkerSize(0.8);
```

Comment:
Suggestion: For the style, in general, you may want to check the original trend plotting macro, since it was updating during the review of the paper, and just re-use the same values,